client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.css

```diff
@@ -0,0 +1,19 @@
+.opened-rma {
+  margin: 20px;
+}
+
+.rma-line {
+  margin-bottom: 10px;
+}
+
+.rma-detail-label {
+  color: #59697D;
+  line-height: 1.5;
+  display: inline-flex;
+  font-size: 1rem;
+  font-weight: 400;
+}
+
+.rma-detail-value {
+  display: inline;
```

Comment:
In my opinion we can get rid of the entire file and use existing helpers and ovh-ui-kit directly.